components/third_party/local-data/LICENSE

```diff
@@ -0,0 +1,1102 @@
+These licenses do not apply to any of the code shipped with the Brave Browser, but may apply to data files downloaded after installation for use with various Brave features. The Brave Browser and such data files are separate and independent works.
```

Comment:
This file is a leftover from before `components/third_party/local-data/` to `components/third_party/local_data` (hyphen to underscore). The real license file, which is autogenerated and excluded from git is `components/third_party/local_data/LICENSE`.